ui/lib/core/addon/components/form-field.hbs

```diff
@@ -261,7 +261,7 @@
           readonly={{this.isReadOnly}}
           autocomplete="off"
           spellcheck="false"
-          value={{or (get @model this.valuePath) @attr.options.defaultValue}}
+          value={{or @attr.options.defaultValue (get @model this.valuePath)}}
```

Comment:
This will always set the value to the passed default if exists even if the model has a value at `valuePath`, so I think we should put this back